simulation/replication/testdata/replication_simulation_activeactive_regional_failover_start_same_wfid_2.yaml

```diff
@@ -0,0 +1,60 @@
+# This file is a replication simulation scenario spec.
+# It is parsed into ReplicationSimulationConfig struct.
+# Replication simulation for this file can be run via ./simulation/replication/run.sh activeactive_regional_failover_start_same_wfid
+# Dynamic config overrides can be set via config/dynamicconfig/replication_simulation_activeactive_regional_failover_start_same_wfid.yml
```

Comment:
update these comments to match the file name. also add corresponding dynamicconfig file